main.cpp

```diff
@@ -189,7 +202,8 @@ int main(int argc, char *argv[])
     qmlRegisterUncreatableType<TranslationManager>("moneroComponents.TranslationManager", 1, 0, "TranslationManager",
                                                    "TranslationManager can't be instantiated directly");
 
-
+    qmlRegisterUncreatableType<WalletKeysFilesModel>("moneroComponents.walletKeysFilesModel", 1, 0, "walletKeysFilesModel",
+                                                   "TranslationManager can't be instantiated directly");
```

Comment:
WalletKeysFilesModel can't be instantiated directly

Reply:
:+1: